File: Lua/db_connect.lua
require "luasql.mysql"

dbname = "test_db"
dbhost = "localhost"
dbuser = "root"
dbpass = "root1234"

env = assert(luasql.mysql())
con = assert(env:connect(dbname, dbuser, dbpass, dbhost))

cur = con:execute("SELECT * FROM test_table1")
row = cur:fetch({}, "a")
while row do
  print(string.format("#%03d %s (%s)", row.id, row.name))
  row = cur:fetch({}, "a")
end

cur:close()
con:close()
env:close()

File: Python/db_connect.py
#!/usr/bin/env python3
# coding: utf-8

import MySQLdb

con = MySQLdb.connect(host='localhost',
                      port=3306,
                      db="test_db",
                      user="root",
                      passwd='root1234',
                      charset='ascii')
cur = con.cursor()
# connected = con.is_connected()
# print(connected)
# if (not connected):
#     con.ping(True)

cur.execute('SELECT * FROM test_table1')
table = cur.fetchall()
print(table)
